network/gossip/libp2p/middleware.go

```diff
@@ -46,6 +46,8 @@ const (
 	DefaultUnicastTimeout = 2 * time.Second
 )
 
+var unicastTimeout = DefaultUnicastTimeout
```

Comment:
Why do we have both a variable and a constant for this?

Reply:
to help it easy to test. In one of my test, I would like to raise this timeout to a high value so that I don't run into context time out issues while a call is blocked.